aspnetcore/signalr/signalr-core-hubs.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Use Hubs in SignalR for ASP.NET Core
+author: rachelappel
+description: Learn how to use hubs in SignalR Core.
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 03/08/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/use-hubs-signalr-core
+---
+
+# Use Hubs in SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel) and [Kevin Griffin](http://twitter.com/1kevgriff)
+
+
+## What is a SignalR Hub
+
+The SignalR Hubs API enables you to call methods on connected clients from the server. In server code, you define methods that can be called by clients, and you call methods that run on the client. In client code, you define methods that can be called from the server, and you call methods that run on the server. SignalR takes care of the client-to-server plumbing for you.
+
+## Configure connections 
+### Connection Ids
+Each connection to a SignalR hub will be given its own connection id.  This connection id can be used by other parts of your application to send messages directly to a particular connection.
```

Comment:
No need to anthropomorphize connections. Avoid future tense How about
Each con to a SignalR  hub has a unique connection id. 
The connection ID can be used in your app to send messages directly to ??? for sure not ~particular~
If you're calling it from your app, you must want to connect to the specified client?

Reply:
This isn't anywhere near ready for review yet and much will change.